.env.sample

```diff
@@ -0,0 +1,9 @@
+SECRET_KEY=nad7kycy!op3$^p@37=xk!mcwmmv5zw6@*e$r+#m5ood2)@i78
+DEBUG=on
+ENGINE=django.db.backends.postgresql
+DB_NAME=skill_finder
```

Comment:
DB_NAME=<DB_NAME> 
Do similar changes for all the variables below this

Reply:
Done